server/utils/reportUtils.test.ts

```diff
@@ -1,9 +1,10 @@
+import { fromPartial } from '@total-typescript/shoehorn'
 import { reportOptions } from './reportUtils'
 
 describe('reportUtils', () => {
   describe('reportOptions', () => {
-    it('should return a list of report options', () => {
-      expect(reportOptions).toEqual([
+    it('should return a list of report options, excluding pii by default', () => {
+      expect(reportOptions(fromPartial({ permissions: [] }))).toEqual([
```

Comment:
A better approach here would be to use the user details factory to be more explicit about the argument being passed, and avoid type mocking:

```suggestion
      const user = userDetailsFactory.build({ permissions: [] })
      expect(reportOptions(user)).toEqual([
```

(This will obviously require the factory imported too, so can't be merged as is 🙂)

Reply:
Done! thanks for the proposed change, made it much easier 👍